model/Aktivita/Aktivita.php

```diff
@@ -748,6 +761,8 @@ public static function editorZpracuj(): ?Aktivita
             $a['zacatek'] = ($zacatekDen)->add(new \DateInterval('PT' . $a['zacatek'] . 'H'))->formatDb();
 
             $konecDen = Program::denAktivityDleKonce($a);
+
+            // TODO: pokud je $a['konec'] 3 a větší, tak se vždycky přičte o hodinu navíc (dělá to fce add)
```

Comment:
TODO zní, jako že funkce add se chová zvláštně pro čísla 3 a větší?

Reply:
@jaroslavtyc Ano, a bylo by to určitě fajn fixnout, je to docela velký bug. Ale vypadá to na něco hodně hluboko. Nenapadá tě, co s tím?